pkgs/development/libraries/libimobiledevice/default.nix

```diff
@@ -2,45 +2,51 @@
 , stdenv
 , fetchFromGitHub
 , autoreconfHook
-, libtool
 , pkg-config
 , gnutls
 , libgcrypt
-, libtasn1
-, glib
 , libplist
+, libtasn1
 , libusbmuxd
+, libimobiledevice-glue
+, SystemConfiguration
+, CoreFoundation
 }:
 
 stdenv.mkDerivation rec {
   pname = "libimobiledevice";
-  version = "unstable-2021-06-02";
+  version = "unstable-2022-05-22";
 
   src = fetchFromGitHub {
-    owner = pname;
+    owner = "libimobiledevice";
     repo = pname;
-    rev = "ca324155f8b33babf907704828c7903608db0aa2";
-    sha256 = "sha256-Q7THwld1+elMJQ14kRnlIJDohFt7MW7JeyIUGC0k52I=";
+    rev = "12394bc7be588be83c352d7441102072a89dd193";
+    hash = "sha256-2K4gZrFnE4hlGlthcKB4n210bTK3+6NY4TYVIoghXJM=";
   };
 
+  # write version including last tagged release so downstream package version
+  # checks will pass
+  postPatch = ''
+    echo '1.3.0-${version}' > .tarball-version
+  '';
+
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [
     autoreconfHook
-    libtool
     pkg-config
   ];
 
   propagatedBuildInputs = [
-    glib
```

Comment:
Does this no longer depend on glib? Check the `Requires` field in the pkg-config file in `${dev}/lib/pkgconfig`.

Reply:
It hasn't for a decade, I'm not sure why this was ever in there. But I double checked the pkg-config and glib is indeed not mentioned.